### ISSUE TITLE
Remove cipher suite and version from Secret

### ISSUE THREAD

**Issue by raphaelrobert** [openmls/openmls#1511]
`Secret` currently also stores the cipher suite and the protocol version:

```rust
pub(crate) struct Secret {
    pub(in crate::ciphersuite) ciphersuite: Ciphersuite,
    pub(in crate::ciphersuite) value: SecretVLBytes,
    pub(in crate::ciphersuite) mls_version: ProtocolVersion,
}
```

 Since the library won't handle different protocol versions we should remove this.
This likely affects the storage format and therefore depends on #672.

**Comment by kkohbrok** [comment]
This was originally done s.t. secrets are not used across versions and ciphersuites. I'm not entirely convinced we should drop this.

**Comment by raphaelrobert** [comment]
We are about to drop protocol version support, so this would only be about the cipher suite. I'm not convinced this is a meaningful measure unless we do this comprehensively and annotate other things like HPKE keys as well. If that's what you want you should open another issue and we discuss the overall strategy there.

**Comment by kkohbrok** [comment]
The fact that the solution is partial doesn't mean we should throw it out alltogether. Of course it would be great to annotate HPKE and signature keys as well. I've opened #1514.
